src/crimson/os/alienstore/CMakeLists.txt

```diff
@@ -74,15 +70,17 @@ if(WITH_LTTNG)
   add_dependencies(crimson-alienstore bluestore-tp)
 endif()
 
+target_compile_options(seastar
+  PUBLIC "-Wno-non-virtual-dtor")
```

Comment:
better now. but this 

- repeats how we set `Seastar_CXX_FLAGS` in `src/CMakeLists.txt`. 
- could be confusing. as we are not building `seastar` here. unfortunately, Seastar does not provide an option allowing Seastar applications to provide "PUBLIC" compile options. i'd move this call into `src/CMakeLists.txt` right after `add_subdirectory(seastar)` , with a comment explaining why we add this option in both places

  * `Seastar_CXX_FLAGS`
  * `target_compile_options(seastar ..)` even we don't build seastar explicitly.

either way, i don't want to block you. will try to come up with a change once yours gets merged.

Reply:
> repeats how we set Seastar_CXX_FLAGS in src/CMakeLists.txt.

This is addressed in next commits.

> either way, i don't want to block you

Would you like to push it here instead? It could make out iterations faster.

Reply:
> > repeats how we set Seastar_CXX_FLAGS in src/CMakeLists.txt.
> 
> This is addressed in next commits.
> 
> > either way, i don't want to block you
> 
> Would you like to push it here instead? It could make out iterations faster. Or all the concerns are already addressed in the next commits.

since it's addressed in the same pull request. probably no need to push it here. i'd squash these two commits if it's fine by you?

Reply:
squashed!